src/compiler/output-targets/dist-custom-elements/index.ts

```diff
@@ -314,7 +319,7 @@ export const generateEntryPoint = (
  * @param outputTarget the output target configuration
  * @returns a list of transformers to use in the transpilation process
  */
-const getCustomElementCustomTransformer = (
+const getBeforeCustomTransformers = (
```

Comment:
should this function possibly be called `getCustomBeforeTransformers`? that would be consistent with the property added to `BundleOptions` I think

Reply:
https://github.com/ionic-team/stencil/pull/4501/commits/fc54d37d4df6de33391442271b2f4828ff5fc2dd